tsconfig.json

```diff
@@ -27,8 +27,8 @@
     // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
 
     /* Additional Checks */
-    "noUnusedLocals": true,                /* Report errors on unused locals. */
-    "noUnusedParameters": true,            /* Report errors on unused parameters. */
+    "noUnusedLocals": false,                /* Report errors on unused locals. */
+    "noUnusedParameters": false,            /* Report errors on unused parameters. */

```

Comment:
Please leave this unchanged in your PR. It is perfectly fine to make this change for testing your local env, but we do want these to be enforced in master. Also I think the branch may be out of sync with the `clean` branch, might want to pull (we aren't using `noUnusedParameters`).

Reply:
You do indeed have conflicts with the base branch to deal with